### PR TITLE
Mempool fillness logging (both for new block and new tx)

### PR DESCRIPTION
**Pull request by 365andreas**

### CODE REVIEW

**Comment by coot** [comment]
@365andreas is there anything else you'd like to add to this PR?

**Comment by 365andreas** [comment]
@coot No I am ok.

File: ouroboros-consensus/demo-playground/Mock/TxSubmission.hs [hunk 1]
```diff
@@ -102,27 +104,34 @@ submitTx n tx = do
...
-readIncomingTx tracer kernel Decoder{..} = forever $ do
+readIncomingTx tracer mempoolTracer kernel Decoder{..} = forever $ do
     newTx :: Mock.Tx <- decodeNext decode
-    rejected <- addTxs (getMempool kernel) [demoMockTx (getNodeConfig kernel) newTx]
+    let memPool = getMempool kernel
+    rejected <- addTxs memPool [demoMockTx (getNodeConfig kernel) newTx]
+    txs <- STM.atomically $ getTxs memPool
+    traceWith tracer $
+      "New incoming tx, txs number in a mempool: " <> show (Seq.length txs)
+    traceWith mempoolTracer $ AfterAddTxs $ Seq.length txs
```
Comment (coot) [inline ouroboros-consensus/demo-playground/Mock/TxSubmission.hs]: @365andreas after merging #659 I think this changes should go to `Ouroboros.Cosensus.TxSubmission`

**Comment by denisshevchenko** [comment]
@CodiePP told me:

> And, please put the "WIP" label again. I put it there on purpose.

Ok, I set it again.

**Comment by dcoutts** [comment]
I think we will need to do this somewhat differently, and do it within the mempool implementation itself.

@intricate can handle that and then close this PR when he's got the alternative approach done.

**Comment by 365andreas** [comment]
implemented by https://github.com/input-output-hk/ouroboros-network/pull/696

**Pull request closed without merging**